components/infobox/wikis/counterstrike/infobox_league_custom.lua

```diff
@@ -215,6 +234,10 @@ function CustomLeague:getWikiCategories(args)
 		table.insert(categories, 'Valve Sponsored Tournaments')
 	end
 
+	if String.isNotEmpty(args.restrictions) then
+		Array.append(categories, Array.map(CustomLeague.getRestrictions(args.restrictions),
```

Comment:
```suggestion
		Array.appendWith(categories, Array.map(CustomLeague.getRestrictions(args.restrictions),
```
so that the categories actually get mutated